src/Quizzer.API.DAL/Entities/Answer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutoMapper;
+
+namespace Quizzer.API.DAL.Entities {
+    public record Answer : EntityBase {
+        public required string Text { get; set; }
+        public string? PictureUrl { get; set; }
+        public required bool IsCorrect { get; set; }
+        public int? Order { get; set; }
+
+        public Guid QusetionId { get; set; }
```

Comment:
Forgot to use navigation property here, Also you have bad spelling here.

Reply:
No i did not i just do not thing it is neccessary to load the whole Question from DB every time you want to see Answer. But it is not a problem to add it. Do we really want it?

Reply:
You are not loading it unless you explicitly specify it in Navigation property paths during the query.